rectools/models/nn/transformer_base.py

```diff
@@ -551,6 +724,19 @@ def _fit(
             val_loss_name=self.val_loss_name,
         )
 
+    def _fit(
+        self,
+        dataset: Dataset,
+    ) -> None:
+        self.data_preparator.process_dataset_train(dataset)
+        train_dataloader = self.data_preparator.get_dataloader_train()
+        val_dataloader = self.data_preparator.get_dataloader_val()
+
+        torch_model = deepcopy(self._torch_model)
+        torch_model.construct_item_net(self.data_preparator.train_dataset)
+
+        self._init_lightning_model(torch_model, self.data_preparator.n_item_extra_tokens)
```

Comment:
`n_item_extra_tokens` isn't used to initialization `lightning_model`

Reply:
fixed